From 806bca8b1312ce9277e937152363bc6d6df24223 Mon Sep 17 00:00:00 2001
From: Priyanka Jain <priyanka.jain@nxp.com>
Date: Mon, 4 Sep 2017 12:28:04 +0530
Subject: [PATCH 141/211] board/ls2080ardb: Add mcmemsize variable in default env

For most of ls2080ardb use-cases, mc private DRAM block is required
to be of 1.75GB.
Hence set  mcmemsize=0x70000000 in default env

Signed-off-by: Priyanka Jain <priyanka.jain@nxp.com>
---
 include/configs/ls2080ardb.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/configs/ls2080ardb.h b/include/configs/ls2080ardb.h
index fb7c794..36c7847 100644
--- a/include/configs/ls2080ardb.h
+++ b/include/configs/ls2080ardb.h
@@ -379,6 +379,7 @@ unsigned long get_board_sys_clk(void);
 	"kernel_addr_sd=0x8000\0"		\
 	"kernel_size_sd=0x14000\0"		\
 	"console=ttyAMA0,38400n8\0"		\
+	"mcmemsize=0x70000000\0"		\
 	BOOTENV					\
 	"boot_scripts=ls2088ardb_boot.scr\0"	\
 	"boot_script_hdr=hdr_ls2088ardb_bs.out\0"	\
-- 
1.7.1

